### PR TITLE
Add support for collecting logs of Spark+K8s jobs

### PR DESCRIPTION
**Pull request by liu-sheng**
Closes-Bug: theopenlab/openlab#210

### CODE REVIEW

**Comment by kiwik** [comment]
Hi @liu-sheng , according Github document, close bug keyword should be following [words](https://help.github.com/en/articles/closing-issues-using-keywords), `Closes-Bug` can not trigger bug close, please update your first comment to follow it

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -1087,6 +1087,8 @@
     description: |
       Run integration tests of spark of master with k8s cluster deployed by kubeadm
     run: playbooks/spark-integration-test-kubeadm-k8s/run.yaml
+    vars:
+      k8s_log_dir: '{{ ansible_user_dir }}/workspace/logs/kubernetes'
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: Whether we can collect kubernetes services log to here, like: apiserver.log?

Comment (kiwik) [reply zuul.d/jobs.yaml]: @liu-sheng sorry, my fault, you collect all of kube-system namespace pod logs, that should include apiserver pod logs, I got it

**Comment by liu-sheng** [comment]
> Hi @liu-sheng , according Github document, close bug keyword should be following [words](https://help.github.com/en/articles/closing-issues-using-keywords), `Closes-Bug` can not trigger bug close, please update your first comment to follow it

thanks, learnt it :)

**Pull request merged**